senza/cli.py

```diff
@@ -1484,6 +1485,11 @@ def scale(stack_ref, region, desired_capacity):
 
     asg = BotoClientProxy('autoscaling', region)
 
+    stack_count = len(list(get_stacks(stack_refs, region)))
+    if not force and stack_count > 1:
+        confirm_str = 'Number of stacks to be scaled - %s. Do you want to continue?' % stack_count
```

Comment:
use [.format](https://docs.python.org/3/library/string.html#string-formatting) instead of the legacy `%` formatting.